package/gluon-mesh-vpn-wireguard/files/lib/netifd/proto/gluon_wireguard.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/sh
+# Copyright 2016-2017 Christof Schulze <christof@christofschulze.com>
+# Licensed to the public under the Apache License 2.0.
+
+. /lib/functions.sh
+. ../netifd-proto.sh
+init_proto "$@"
+
+proto_gluon_wireguard_init_config() {
+  no_device=1
+  available=1
+  renew_handler=1
+}
+
+proto_gluon_wireguard_renew() {
+  local config="$1"
+  echo wireguard RENEW: $@
+  ifdown $config
+  ifup $config
```

Comment:
renew could be improved by using `wg setconf`, except I guess we're missing a proper config file for that, right?

Reply:
right.